testament/specs.nim

```diff
@@ -364,7 +366,18 @@ proc parseSpec*(filename: string): TSpec =
         of "netbsd":
           when defined(netbsd): result.err = reDisabled
         else:
-          result.parseErrors.addLine "cannot interpret as a bool: ", e.value
+          block checkHost:
+            for os in platform.OS:
+              if value == os.name.normalize:
+                if value == hostOS.normalize:
```

Comment:
Rewrite the logic or add a comment, so in the end hostOS should be the same as os.name? Why?

Reply:
See new code comments